### PR TITLE
chore(repo): updated husky to v9.1.5

### PR DESCRIPTION
**Pull request by Phillip9587**
## Current Behavior

`husky@v8`

## Expected Behavior

`husky@v9`

## Related Issue(s)

Fixes #

### CODE REVIEW

**Comment by Phillip9587** [comment]
CI Failure is unrelated to the change

File: .husky/pre-push [hunk 1]
```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/env sh
-
 pnpm check-lock-files &&
-pnpm check-commit &&
-pnpm documentation &&
-pnpm pretty-quick --check
+    pnpm check-commit &&
```
Comment (JamesHenry) [inline .husky/pre-push:2]: Why this indentation?

Comment (JamesHenry) [reply .husky/pre-push:2]: Why do we even have ampersands here? 🤔 

Let's remove them and just have individual lines I think

**Review by JamesHenry — CHANGES_REQUESTED** [review]
See comments

File: .husky/pre-push (hunk 1 above)
Comment (Phillip9587) [reply .husky/pre-push]: Done

**Review by JamesHenry — APPROVED** [review]

**Pull request merged**